--- origsrc/exiv2-0.27.4/tests/bash_tests/testcases.py	2021-06-15 22:57:33.000000000 +0900
+++ src/exiv2-0.27.4/tests/bash_tests/testcases.py
@@ -885,9 +885,9 @@ set Exif.Photo.DateTimeDigitized 2020:05
         else:
             LANG = 'LANG'
 
-        share_dir    = os.path.normpath(os.path.join(BT.Config.bin_dir, '..', 'share2'))
+        share_dir    = os.path.normpath(os.path.join(BT.Config.bin_dir, '..', 'share'))
         os.makedirs(share_dir, exist_ok=True)
-        locale_dir   = '/usr/local/share/locale'
+        locale_dir   = '../../../../inst/usr/share/locale'
         if os.path.isdir(locale_dir) and os.path.isdir(share_dir):
             BT.cp(locale_dir, share_dir)
         else:
@@ -895,7 +895,7 @@ set Exif.Photo.DateTimeDigitized 2020:05
 
         # The above part is checking the environment, and the following part is executing the actual test
         out      = BT.Output()
-        for language in ['fr_FR', 'es_ES']:
+        for language in ['fr_FR.utf8', 'es_ES.utf8']:
             e    = BT.Executer(cmd='exiv2', extra_env={'LC_ALL': language, LANG: language}, assert_returncode=[1], decode_output=False)
             out += e.stdout.split(b'\n')[0].decode()
         BT.reportTest('nls-test', out,forgive=True)
