--- origsrc/exiv2-0.17.1/src/Makefile	2008-04-14 22:46:01.000000000 -0500
+++ src/exiv2-0.17.1/src/Makefile	2008-06-22 04:28:38.468750000 -0500
@@ -237,7 +237,7 @@
 bin: lib $(BINARY) $(EXIV2BIN) $(MCBIN) path-test
 
 lib: $(OBJ)
-	$(LIBTOOL) --mode=link $(LINK.cc) -o $(LIBRARY) $(LOBJ) -rpath $(libdir) -version-info $(EXIV2_LTVERSION) $(LIBS)
+	$(LIBTOOL) --mode=link $(LINK.cc) -o $(LIBRARY) $(LOBJ) -rpath $(libdir) -version-info $(EXIV2_LTVERSION) -no-undefined $(LIBS)
 	@touch lib
 
 path-test: path-test.o utils.o
